### PR TITLE
Meson g12

### PR DESCRIPTION
**Pull request by touilkhouloud**

### CODE REVIEW

**Comment by touilkhouloud** [comment]
Sent pull request for LAVA device-type: https://git.lavasoftware.org/lava/lava/merge_requests/445

File: test-configs.yaml [hunk 1]
```diff
@@ -461,6 +461,24 @@ device_types:
...
+    flags: ['little_endian']
+    filters:
+      - blacklist: {defconfig: ['allnoconfig', 'allmodconfig']}
+
+  meson-g12a-x96-max:
+    name: 'meson-g12a-x96-max'
+    mach: amlogic
+    class: arm64-dtb
+    boot_method: uboot
+    flags: ['little_endian']
```
Comment (gctucker) [inline test-configs.yaml]: `little_endian` is not used as a flag anywhere, it doesn't really hurt to add it but it's assumed to be little endian by default on arm* platforms unless `big_endian` is specified.  So it's probably better to not introduce `little_endian` now, and just remove the `flags:` attribute.

Comment (touilkhouloud) [reply test-configs.yaml]: Will apply these changes.

**Comment by gctucker** [comment]
@touilkhouloud I don't see these devices upstream in LAVA:
https://git.lavasoftware.org/lava/lava/tree/master/lava_scheduler_app/tests/device-types

It's a requirement before using new LAVA device types in KernelCI to have them upstream.  Could you please confirm when that has been done?  We shouldn't merge this PR until then.

**Comment by gctucker** [comment]
Hi @touilkhouloud, could you please create a new PR in https://github.com/kernelci/kernelci-core/ as we're shutting down this `kernelci-core-staging` repo this week?

**Comment by touilkhouloud** [comment]
Hello @gctucker  yes for sure.

**Comment by gctucker** [comment]
Closing this PR as there's a new one in the main repo: https://github.com/kernelci/kernelci-core/pull/29

**Pull request closed without merging**